examples/autocomplete/src/App-Mentions.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Mention from 'antd/lib/mention';
+import algoliasearch from 'algoliasearch/lite';
```

Comment:
We have to add `algoliasearch` as dependencies.

Reply:
Good call!